exposed-kotlin-datetime/src/test/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/MiscTableTest.kt

```diff
@@ -1267,6 +1251,23 @@ class MiscTableTest : DatabaseTestsBase() {
     }
 }
 
+object Misc : MiscTable() {
```

Comment:
I simply moved this to the bottom of the file to make it easier to immediately run tests without having to scroll down to find `class MiscTableTest`.